docs/content/concepts/ops-jobs-graphs/graphs.mdx

```diff
@@ -0,0 +1,375 @@
+---
+title: Graphs | Dagster
+description: Graphs are sets of interconnected ops or sub-graphs and form the core of jobs.
+---
+
+# Graphs
+
+A graph is a set of interconnected [ops](/concepts/ops-jobs-graphs/ops) or sub-graphs. While individual ops typically perform simple tasks, ops can be assembled into a graph to accomplish complex tasks.
```

Comment:
I think it's worth mentioning somewhere on this page that you can create a job out of a graph (and linking to the section on the job page). Maybe it's already mentioned and I missed it.